modules/tensor_mechanics/doc/content/source/materials/ADRankTwoDirectionalComponent.md

```diff
@@ -0,0 +1,28 @@
+# Rank Two Directional Component
+
+!syntax description /Materials/ADRankTwoDirectionalComponent
+
+## Description
+
+This is an ADMaterial model used to extract components of a rank-2 tensor in a
+Cartesian coordinate system based on an input direction. This can be used
+regardless of the coordinate system used by the model.
+
+This ADMaterial model is used by
+[TensorMechanics/Master](/Modules/TensorMechanics/Master/index.md)
+automatically, but can also be called directly by the user. This class
+calculates the scalar value of a Rank-2 tensor, $T$, in the direction selected
```

Comment:
Make the same tweaks as for the other ones. 

Also, change 'calculates the scalar value' to 'calculates the component'